pkg/reaper/podreaper/types.go

```diff
@@ -40,3 +42,15 @@ type Args struct {
 	SoftReap      bool
 	DryRun        bool
 }
+
+// FakePod is the pod struct used for testing
+type FakePod struct {
```

Comment:
Why do we need this moved to `types.go` it's the same package, we can keep this code under `podreaper_test.go`